Mathlib/Algebra/Group/Pointwise/Set/Card.lean

```diff
@@ -64,7 +64,8 @@ attribute [deprecated natCard_inv (since := "2024-09-30")] card_inv
 attribute [deprecated natCard_neg (since := "2024-09-30")] card_neg
 
 @[to_additive (attr := simp)]
-lemma encard_inv (s : Set G) : s⁻¹.encard = s.encard := by simp [encard, ENat.card]
```

Comment:
I would guess that replacing `encard` with  `← toENat_cardinalMk` is sufficient to avoid changing the proof.